src/Raven.Client/Constants.cs

```diff
@@ -88,6 +88,13 @@ private Certificates()
             }
 
             public const string Prefix = "certificates/";
+            public const string Certificate = "Certificate";
+            public const string Thumbprint = "Thumbprint";
+            public const string OldThumbprint = "OldThumbprint";
+            public const string Confirmations = "Confirmations";
+            public const string Replaced = "Replaced";
+            public const string ReplaceImmediately = "ReplaceImmediately";
+            public const string CertReplaceAlertTitle = "Server Certificate Replacement";
```

Comment:
Please remove this from Constants. This is not a place for 'property names' and 'titles'.